lib/src/runner/browser/content_shell.dart

```diff
@@ -59,9 +60,16 @@ class ContentShell extends Browser {
         // make sure it's not expired and that the remote debugging port worked.
         // Any errors from this will always come before the "Running without
         // renderer sandbox" message.
-        while (await stderr.moveNext() &&
+        while (await stderr
+                .moveNext()
+                .timeout(_errorTimeout)
+                .catchError((_) => false) &&
             !stderr.current.endsWith("Running without renderer sandbox") &&
-            !stderr.current.contains("Running without the SUID sandbox")) {
+            !stderr.current.contains("Running without the SUID sandbox") &&
+            // Error messages on Mac can get gobbled, we must assume that it
+            // it started up successfully.
```

Comment:
Discussed offline. "kqueue" is the only type of message we receive for content-shell version 50 on Mac. This creates various issues. Adding timeout logic with an application exception encouraging users to make sure they are using the most recent version of content-shell.